acceptance/openstack/rds/v3/flavors_test.go

```diff
@@ -14,12 +14,11 @@ func TestFlavorsList(t *testing.T) {
 	th.AssertNoErr(t, err)
 
 	listOpts := flavors.ListOpts{
-		VersionName: "10",
+		DatabaseName: "PostgreSQL",
+		VersionName:  "10",
 	}
-	allFlavorPages, err := flavors.List(client, listOpts, "PostgreSQL").AllPages()
-	th.AssertNoErr(t, err)
 
-	rdsFlavors, err := flavors.ExtractDbFlavors(allFlavorPages)
+	rdsFlavors, err := flavors.ListFlavors(client, listOpts)
 	th.AssertNoErr(t, err)
```

Comment:
flavors_test.go:22: Failure in flavors_test.go, line 22: unexpected error "json: cannot unmarshal array into Go struct field Flavor.version_name of type string"